pkgs/os-specific/linux/syslinux/default.nix

```diff
@@ -1,14 +1,16 @@
-{ stdenv, fetchFromRepoOrCz, fetchurl, nasm, perl, python3, libuuid, mtools, makeWrapper }:
+{ stdenv, fetchgit, fetchurl, fetchpatch, nasm, perl, python3, libuuid, mtools, makeWrapper }:
 
 stdenv.mkDerivation {
-  name = "syslinux-2019-02-07";
+  pname = "syslinux";
+  version = "2019-02-07";
```

Comment:
Sorry, last thing, the version should be `version = "unstable-20190207";`

Reply:
Should I maybe put these changes in a separate commit?

I agree that we might want to make these changes, while we're at it, but they seem pretty unrelated to the UEFI stuff.